### PR TITLE
export defaultGetLocalIdent

### PR DESCRIPTION
**Pull request by SkReD**
This PR contains a:

- [ ] **bugfix**
- [x] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case
https://github.com/webpack-contrib/css-loader/issues/1422

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -324,7 +324,7 @@ function escapeLocalIdent(localident) {
   );
 }
 
-function defaultGetLocalIdent(
+export function defaultGetLocalIdent(
```
Comment (alexander-akait) [inline src/utils.js:327]: We should not export it here, we should reexport it from the entry point, i.e. `index.js`

Comment (SkReD) [reply src/utils.js:327]: It is done here https://github.com/webpack-contrib/css-loader/pull/1423/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R30

Comment (alexander-akait) [reply src/utils.js:327]: We should not export it from `utils.js`, we can rename file or move it to another places, just reexport it from `index.js`

Comment (SkReD) [reply src/utils.js:327]: This is exactly what i did. But re-export in index it should be exported from utils too

Comment (SkReD) [reply src/utils.js]: ready to review. Sorry for the mess

**Review by alexander-akait — APPROVED** [review]

**Pull request merged**